docs/source/index.rst

```diff
@@ -0,0 +1,15 @@
+Welcome to Emissions API's documentation!
+=========================================
```

Comment:
This page is quite empty and lacks some documentation about the project. To not write this twice, we could include the reproject's description from the [README.md](https://github.com/emissions-api/emissions-api/blob/master/README.md) for which we could also use reStructuredText. Thoughts?

Reply:
Nice idea. Maybe we should create a separate issue for that. Also there seem to be multiple plugins for parsing markdown files in sphinx.

Reply:
We can also simply use reStructuredText for the readme.rst. We have it in the project anyway and GitHub can work with it in the same way it can work with Markdown. But yes, let's move that to a separate issue.